sources/web/datalab/server.ts

```diff
@@ -275,7 +275,7 @@ function uncheckedRequestHandler(request: http.ServerRequest, response: http.Ser
       urlpath.indexOf('/oauthcallback') == 0) {
     // Start or return from auth flow.
     auth.handleAuthFlow(request, response, parsed_url, appSettings);
-  } else if (isStaticResource(urlpath)) {
+  } else if (!reverseProxy.isReverseProxyRequest(request) && isStaticResource(urlpath)) {
```

Comment:
If you move the check here, then you don't need to handle it again in https://github.com/yebrahim/datalab/blob/master/sources/web/datalab/server.ts#L148?